src/flb_input.c

```diff
@@ -108,6 +108,21 @@ struct flb_config_map input_global_properties[] = {
         "pause until the buffer is drained. The value is in bytes. If set to 0, the buffer limit is disabled."\
         "Note that if the plugin has enabled filesystem buffering, this limit will not apply."
     },
+    {
+        FLB_CONFIG_MAP_STR, "storage.type", NULL,
```

Comment:
the default for `storage.type` is `memory`, so I think it should be safe to add it